MIT_LICENSE.TXT

```diff
@@ -0,0 +1,29 @@
+===============================================================
+Modifications to the LLVM Project for OpenCilk are licensed under the
+MIT License with the OpenCilk Addendum:
+===============================================================
+
+Copyright (c) 2020 Massachusetts Institute of Technology
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal with the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+DEALINGS IN THE SOFTWARE.
+
+---- OpenCilk Addendum to the MIT License ----
+
+As an alternative to distributing the Software under this license, you may
+distribute the Software under the LLVM license.
```

Comment:
@neboat As a downstream project who is interested in using this code, this seems a bit ambiguous: First, presumably by "LLVM license" you mean the new Apache 2.0 license w/ LLVM exceptions, not the NCSA/UIUC legacy LLVM license? The former would be useful, as I'd like to be able to keep things simple and distribute my entire downstream codebase (LLVM + Tapir/OpenCilk modifications + Swarm modifications) under one license, which looks like it's going to be Apache 2.0 w/ LLVM exceptions.

Another note: LLVM's Apache 2.0 license Section 4(b) provides this condition on redistribution:
> You must cause any modified files to carry prominent notices
> stating that You changed the files

So it might be helpful if there was a short header comment added at the top of any file that exists in the original LLVM project and that was modified for OpenCilk. If you'd like, I could help with this, by just looking at the diff between this and upstream LLVM and inject a line at the top of the relevant files.